plugin/builtin/legacyordertransformer/go.mod

```diff
@@ -4,7 +4,9 @@ go 1.16
 
 require (
 	github.com/pkg/errors v0.9.1
-	sigs.k8s.io/kustomize/api v0.8.6
+	sigs.k8s.io/kustomize/api v0.0.0-00010101000000-000000000000
```

Comment:
FWIW, this line should be left unchanged.
The replace below doesn't care about the version specified here.